src/api256.rs

```diff
@@ -396,3 +403,37 @@ pub fn subtractPrivateKeys(private_key_a: &[u8], private_key_b: &[u8]) -> Result
     let pubKeyB = PrivateKey::new(util::slice_to_fixed_32_bytes(private_key_b, "privateKeyB"));
     Ok(pubKeyA.augment_minus(&pubKeyB).bytes().to_vec())
 }
+
+/**
+ * Hashes all possible tri-grams for the given string. The values will be prefixed with the partition_id and salt before being hashed.
+ */
+#[wasm_bindgen]
+pub fn generateHashesForString(
+    s: String,
+    partition_id: Option<String>,
+    salt: &[u8],
```

Comment:
I'd maybe change this to take `s, salt, partition_id` instead so that the last argument is optional. Won't change much here, but it'd make it more JavaScript-y to make the last argument optional. Ditto with the method below.

Reply:
Sure, made the change